core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3002,7 +3002,23 @@ init_llvm_jit_functions_stage1(WASMModule *module, char *error_buf,
     if (module->function_count == 0)
         return true;
 
-#if WASM_ENABLE_FAST_JIT != 0 && WASM_ENABLE_LLVM_JIT != 0
+#if WASM_ENABLE_DYNAMIC_PGO != 0
+    if (os_mutex_init(&module->orcjit_stop_lock) != 0) {
+        set_error_buf(error_buf, error_buf_size,
+                      "init orcjit stop lock failed");
+        return false;
+    }
+    if (os_cond_init(&module->orcjit_stop_cond) != 0) {
+        set_error_buf(error_buf, error_buf_size,
+                      "init orcjit stop cond failed");
+        os_mutex_destroy(&module->orcjit_stop_lock);
+        return false;
+    }
+    module->orcjit_stop_lock_inited = true;
+#endif
+
+#if WASM_ENABLE_FAST_JIT != 0 && WASM_ENABLE_JIT != 0 \
```

Comment:
No need to add `WASM_ENABLE_JIT != 0` again, since it is already added in L2992.